### ISSUE TITLE
Investigate use of anchor elements with button classes for accessibility

### ISSUE THREAD

**Issue by stuartmccoll** [heritagefund/funding-frontend#140]
We've used anchor elements in a few places which have a `role` attribute of `button` and are styled using the `govuk-button` class. We should confirm whether this is less accessible than having a `button` element inside a `form` element, and make any changes necessary to ensure that these pages are accessible.

See below for an example:

```
<a href="<%= "#{three_to_ten_k_project_volunteers_url}" %>" role="button" draggable="false" class="govuk-button"
   data-module="govuk-button" aria-label="Continue button">
  Save and continue
</a>
```

**Comment by naps62** [comment]
Admittedly, I'm not an accessibility expert, but `role="button"` should be sufficient if you want it to be interpreted as a button for a screen reader
